README.md

```diff
@@ -7,3 +7,18 @@ Please visit the main project page for more information.
 
 The assets under the `ibls/` folder were sourced from [Deviantart](https://www.deviantart.com/zbyg/art/HDRi-Pack-2-103458406), and are available under [CC BY 3.0 license](LICENSE).
 The images were converted into C headers using [bin2c](https://sourceforge.net/projects/bin2c/).
+
+The colormaps in the `colormaps/` folder have the following licenses:
+
+| Colormap | License | Source |
+|:--:|:--:|--|
+| coolwarm | [CC0] | https://creativecommons.org/choose/zero/ |
```

Comment:
The source url for coolwarm and turbo is incorrect. Also, I think coolwarm is under 3-Clause BSD.